README.md

```diff
@@ -818,6 +819,12 @@ to change Zappa's behavior. Use these at your own risk!
 ```javascript
  {
     "dev": {
+        "alb_enabled": false // enable provisioning of application load balancing resources
```

Comment:
This should also note that the "alb_vpc_config" section is mandatory if alb_enabled is true.